src/app/static/src/js/components/runWorkflow/runWorkflowTabs.vue

```diff
@@ -11,11 +11,11 @@
                         <div class="d-md-block mt-4 mt-md-0 collapse navbar-collapse" id="sidebar">
                             <ul class="nav flex-column list-unstyled mb-0">
                                 <li class="nav-item">
-                                    <a id="run-workflow-link" class="nav-link active" data-toggle="tab" role="tab" href="#"
+                                    <a id="run-workflow-link" class="nav-link" :class="{active: selectedTab == 'runWorkflow'}" data-toggle="tab" role="tab" href="#"
```

Comment:
Don't need type coercion here
```suggestion
                                    <a id="run-workflow-link" class="nav-link" :class="{active: selectedTab === 'runWorkflow'}" data-toggle="tab" role="tab" href="#"
```

Reply:
There's a bit of duplicated code here in terms of making these tabs active, which is also in https://github.com/vimc/orderly-web/pull/366 (which is awaiting your review when you get to it @Nicolas-Dolan ...)